server/server.go

```diff
@@ -1,113 +1,52 @@
 package server
 
-import (
-	"path/filepath"
-
-	pb "github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
-	"github.com/opencontainers/ocitools/generate"
-	"golang.org/x/net/context"
+const (
+	runtimeAPIVersion = "v1alpha1"
```

Comment:
> v1alpha1 was originally set to be the runc's API version.

mmm not sure it makes much sense - runtimes should adhere to v1alpha1 not the other way around afaict /cc @mrunalp